### ISSUE TITLE
[Bug]: Manual Fingerprint Editor doesn’t load Seasons

### ISSUE THREAD

**Issue by sannidhyaroy** [jumoog/intro-skipper#227]
### Self service debugging

- [X] Jellyfin is updated and my permissions are correct (or I did not use Docker)

### Describe the bug

Under `Dashboard > Plugins > Intro Skipper > Manage Fingerprints`, selecting the Series, doesn’t load the seasons, so I’m not able to select any season for the show, thus rendering the fingerprint editor broken.

I've been having this issue for a very long time, so I think this bug has been there for a long time. I used to use ConfusedPolarBear's Intro Skipper before, until the repo was archived and didn’t recall having such an issue, back then.

Running Jellyfin v10.9.8 on a Raspberry Pi 4 Model B (8 GB RAM) [on DietPi v9.6.1 based on Raspberry Pi OS]

### Jellyfin install method

DietPi Software (uses apt under the hood)

### Container image/tag or Jellyfin version

Jellyfin v10.9.8, Intro Skipper v0.2.0.13

### Operating System

DietPi v9.6.1 (based on Raspberry Pi OS)

### IMDb ID of that TV Series

Applicable to every series (Not dependent on particular show)

### Support Bundle

```shell
* Jellyfin version: 10.9.8
* Plugin version: 0.2.0+11e648ffbbeb
* Queue contents: 173 episodes, 13 seasons
* Warnings: `None`
* FFmpeg: `okay`

FFmpeg version:

ffmpeg version 6.0.1-Jellyfin Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 12 (Debian 12.2.0-14)
configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-static --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-shared --enable-gmp --enable-gnutls --enable-chromaprint --enable-opencl --enable-libdrm --enable-libxml2 --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libdav1d --enable-libsvtav1 --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=arm64 --cross-prefix=/usr/bin/aarch64-linux-gnu- --toolchain=hardened --enable-cross-compile --enable-rkmpp --enable-rkrga
libavutil      58.  2.100 / 58.  2.100
libavcodec     60.  3.100 / 60.  3.100
libavformat    60.  3.100 / 60.  3.100
libavdevice    60.  1.100 / 60.  1.100
libavfilter     9.  3.100 /  9.  3.100
libswscale      7.  1.100 /  7.  1.100
libswresample   4. 10.100 /  4. 10.100
libpostproc    57.  1.100 / 57.  1.100
```
```


### Jellyfin logs

_No response_

**Comment by AbandonedCart** [comment]
This is actively being rewritten, so it may be best to hold off until the pending changes are finished to investigate if this is still an issue.

**Comment by jumoog** [comment]
Please test this version: https://github.com/jumoog/intro-skipper/releases/tag/10.9%2Fpreview and let me know if the problem still exists. You have to replace the dll manually and restart jellyfin.

**Comment by sannidhyaroy** [comment]
> Please test this version: https://github.com/jumoog/intro-skipper/releases/tag/10.9%2Fpreview and let me know if the problem still exists. You have to replace the dll manually and restart jellyfin.

No, even the redesigned fingerprint editor in this pre-release doesn’t solve the issue.
Also, I should have mentioned that before opening this issue, I removed the plugin, removed the configurations and `.xml` files where the timestamps are stored, and then reinstalled it to start fresh and check if the issue still persists, and it didn’t fix it.

https://github.com/user-attachments/assets/5dfb0f13-735d-40fe-93db-28f20db98ddc

**Comment by jumoog** [comment]
thank you for the video

**Comment by sannidhyaroy** [comment]
This is an issue with the reverse proxy. I've faced this issue for a long time, yet it never stuck me but now when I tested it with the IP Address and port, no such issue is experienced.

My Nginx config is according to the [Jellyfin Documentation](https://jellyfin.org/docs/general/networking/nginx). After some tinkering, removing `permission policy` line seems to fix the issue. This line is only included if  `Nginx from a subdomain (jellyfin.example.org)` is your type of accessing your Jellyfin instance.

### TLDR:
1. Issue is caused by the reverse proxy.
2. Open your nginx config and uncomment or remove the line containing `add_header Permissions-Policy "accelerometer=(), ambient-light-sensor=(), battery=(), bluetooth=(), camera=(), clipboard-read=(), display-capture=(), document-domain=(), encrypted-media=(), gamepad=(), geolocation=(), gyroscope=(), hid=(), idle-detection=(), interest-cohort=(), keyboard-map=(), local-fonts=(), magnetometer=(), microphone=(), payment=(), publickey-credentials-get=(), serial=(), sync-xhr=(), usb=(), xr-spatial-tracking=()" always;`.
3. Reload the reverse proxy.
